src/test/rgw/test_rgw_aws.cc

```diff
@@ -0,0 +1,56 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include <gtest/gtest.h>
+#include <aws/core/Aws.h>
+#include <rgw/rgw_aws.h>
+#include <chrono>
+#include <thread>
+#include "common/ceph_context.h"
+#include <aws/core/auth/AWSCredentialsProvider.h>
+
+class TestAWS : public ::testing::Test {
+protected:
+
+  void SetUp() override {
+    auto cct = new CephContext(CEPH_ENTITY_TYPE_CLIENT);
+    ASSERT_TRUE(rgw::aws::init(cct));
+  }
+
+  void TearDown() override {
+    rgw::aws::shutdown();
+  }
+
+};
+void callback(int status){
+  if(status == 1){
+    std::cout << "Successful" << std::endl;
+  }else{
+    std::cout << "Failed" << std::endl;
+  }
+
+}
+
```

Comment:
please add documentation in comment:
* install moto
* install docker
* create the lambda function (please provide the actual `aws` cli command that matches the code)
* create the sns topic (please provide the actual `aws` cli command that matches the code)

Reply:
* please add a comment on that region has to be a valid aws region (e.g. "us-east-1")